cmd/cli/serve/util.go

```diff
@@ -162,12 +162,12 @@ func SetupIPFSClient(ctx context.Context, cm *system.CleanupManager, ipfsCfg typ
 	return client, nil
 }
 
-func getDisabledFeatures() (node.FeatureConfig, error) {
-	var featureConfig node.FeatureConfig
-	if err := config.ForKey(types.NodeDisabledFeatures, &featureConfig); err != nil {
-		return node.FeatureConfig{}, err
+func getConfig[T any](name string) (T, error) {
+	var obj T
+	if err := config.ForKey(name, &obj); err != nil {
+		return obj, err
 	}
-	return featureConfig, nil
+	return obj, nil
 }
```

Comment:
Nice! Is there any reason to use this over `config.Get`? Perhaps we should modify the existing `Get` method?

Reply:
Yeah, not sure why this was here really. `Get` should work fine!